### DISCUSSION TITLE
Is there a built-in function to get custom_fields from a post in bS Grid

### DISCUSSION THREAD

**Discussion by rotkppchn** [bootscore/bootscore#570] [Q&A] [answered]
I discovered your theme today while searching for a (basic) Bootstrap WP theme and would like to output a list of posts on a static page via the bS Grid plugin. This already works, but I would like to display three bulletpoints at the point of the template where the post content comes, ideally via custom fields from the post. Is there already a built-in function similar to bootscore_author(), bootscore_date(), etc... that I have seen for this that simply accesses custom fields within the loop?

**Comment by crftwrk** [comment, marked as answer]
> Is there already a built-in function similar to bootscore_author(), bootscore_date(), etc... 

No, but you can use [Advanced Custom Fields](https://wordpress.org/plugins/advanced-custom-fields/) to create quickly fields and add them to the bS Grid template.
